data/changelog.txt

```diff
@@ -543,7 +543,7 @@ Obsolete flaming weapons.
 Obsolete survivor helmets.
 Realistic gun volume campaign.
 Migrates+obsoletes firearms incongruent with lore.
-Tameable Piglets.
```

Comment:
Both of these are valid spellings, so it'd be better not to change it.

Reply:
Good catch. Reverted this (and 3 other locations) to tameable